config/settings.yml

```diff
@@ -358,6 +358,7 @@ claims_api:
 redis:
   host: localhost
   port: 6379
+  duplicate_to_secondary: false
```

Comment:
Is having the 2nd set of URLs (`secondary_redis`) just as effective as the flag? Seems like its one less thing to worry about if you look for the config instead of the flag _and_ the config.

Reply:
Opened the ops PR here: https://github.com/department-of-veterans-affairs/devops/pull/6877